server_admin/topics/sso-protocols/con-saml.adoc

```diff
@@ -0,0 +1,13 @@
+[id="con-saml_{context}"]
+
+[[_saml]]
```

Comment:
I think there may be a problem with using two anchor ids.  Using CCutil on this caused an error, so best to remove it here.